common/src/device/transport/mqtt/ClientWebSocketChannel.cs

```diff
@@ -83,6 +82,15 @@ public ClientWebSocketChannel Option<T>(ChannelOption<T> option, T value)
             return this;
         }
 
+        public void Dispose()
+        {
+            _webSocket?.Dispose();
+            _webSocket = null;
```

Comment:
Do we really need to set to null after dispose?

Reply:
Do we have to? No, not in this case. I do it as a best practice for any disposable object so we never accidentally try to dispose twice.